### PR TITLE
Enhancing mem-bw testcase to print individual bank name and throughput.

### PR DESCRIPTION
**Pull request by ManojTakasi**

### CODE REVIEW

File: tests/validate/bandwidth_test/src/host.cpp [hunk 1]
```diff
@@ -87,7 +88,15 @@ int main(int argc, char** argv) {
             if (sValue == "HBM") {
                 chk_hbm_mem = true;
             }
-        }
+	    else if (sValue == "DDR") {
+		    auto banks=pt_mem_entry.get_child("banks");
+	            for (const auto&bank:banks) {
+			auto bank_name=bank.second.get<std::string>("name");
```
Comment (saifuddin-xilinx) [inline tests/validate/bandwidth_test/src/host.cpp]: Use space. This is applicable for all the lines

File: tests/validate/bandwidth_test/src/host.cpp [hunk 2]
```diff
@@ -72,6 +72,7 @@ int main(int argc, char** argv) {
     bool chk_hbm_mem = false;
     std::string filename = "/platform.json";
     auto platform_json = boost::filesystem::path(test_path) / filename;
+    std::vector<std::string>bank_names;
```
Comment (saifuddin-xilinx) [inline tests/validate/bandwidth_test/src/host.cpp]: Add space before variable name

File: tests/validate/bandwidth_test/src/host.cpp [hunk 3]
```diff
@@ -161,6 +171,8 @@ int main(int argc, char** argv) {
     }
 
     double max_throughput = 0;
+    std::vector<double> bank_throughput_values(num_kernel_ddr,0.0);
```
Comment (saifuddin-xilinx) [inline tests/validate/bandwidth_test/src/host.cpp]: Same

**Review by saifuddin-xilinx — CHANGES_REQUESTED** [review]

**Comment by ManojTakasi** [comment]
@saifuddin-xilinx changes has been added according your comments. Could you please let me know any other changes are required.

**Pull request closed without merging**